atat_provisioning_wizard_api.yaml

```diff
@@ -1061,24 +1070,47 @@ components:
       type: object
       additionalProperties: false
       properties:
-        first_name:
-          type: string
-        last_name:
+        display_name:
           type: string
         email:
           type: string
           format: "email"
-        access:
-          description: Operator Access Level to Environment
-          type: string
-          enum:
-            - administrator
-            - read_only
       required:
         - first_name
         - last_name
         - email
+    PortfolioOperator:
+      allOf:
+        - $ref: '#/components/schemas/Operator'
+      type: object
+      additionalProperties: false
+      required:
+        - access
+      properties:
+        access:
+          $ref: '#/components/schemas/PortfolioAccess'
+    AppEnvOperator:
+      allOf:
+        - $ref: '#/components/schemas/Operator'
+      type: object
+      additionalProperties: false
+      required:
         - access
+      properties:
+        access:
+          $ref: '#/components/schemas/AppEnvAccess'
+    PortfolioAccess:
+      description: Represents a set of Operators who should be granted access to a Portfolio at a specific access level. Uses an enum construct for forward compatibility should other access levels be specified.
+      type: string
+      enum:
+        - root_administrator
+    AppEnvAccess:
+      description: Represents a set of Operators who should be granted access to an Application or Environment at a specific access level. Is a single type because the same access levels are available for both Applications & Environments.
+      type: string
+      enum:
+        - administrator
+        - contributor
+        - read_only
```

Comment:
Do we need a `no_access` level?  Eric was suggesting at the top of today's call that we might have a need to store that.